examples/enterprise_edr/device_processes.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+import sys
+import logging
+import concurrent.futures
+from cbc_sdk.helpers import eprint, build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import Device
+from cbc_sdk.enterprise_edr import Process
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    parser = build_cli_parser()
+
+    args = parser.parse_args()
+    cb = get_cb_cloud_object(args)
+
+    query = cb.select(Device).where('1')
```

Comment:
Let's make this a query for active devices using "ACTIVE" for device_status

Reply:
How do you write that? Is it just "device_status:ACTIVE"?

Reply:
It's a criteria property https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/devices-api/

Reply:
https://github.com/carbonblack/carbon-black-cloud-sdk-python/blob/develop/src/cbc_sdk/platform/devices.py#L335

Reply:
OK, I remember now.  Adding the set_status() to the query reduced the number of devices being queried from 116 to 15.